spec/support/govuk_i18n_cov.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class GOVUKI18nCov
+  module I18n
+    module Registry
+      def lookup(locale, key, scope = [], options = {})
+        GOVUKI18nCov.log(locale, key, scope)
+        super
+      end
+    end
+  end
+
+  class << self
+    def start
+      return if @started
+
+      @started = true
+      @used_keys = []
+      ::I18n::Backend::Simple.send :include, I18n::Registry
```

Comment:
Looks like include is a public method: https://github.com/ruby-i18n/i18n/blob/0c5dab494d9b043e00662d8e789229c33045c024/lib/i18n/backend/simple.rb#L24